joomla/cassiopeia
# Remove *.* selector from _forms.scss

Pull Request for Issue #106 .

### Testing Instructions
Run npm ci or compile scss

Check all forms present on the page after login on frontend.
Check a compact list of tags (create a menu item for this)

### Expected result
The buttons on a compact list of tags are aligned
Other forms are not affected and the spacing between fieldsets and between elements inside a fieldset is still correct.
![grafik](https://user-images.githubusercontent.com/9153168/94564051-caaaa480-0267-11eb-9198-42083a0b2667.png)


### Actual result
The buttons on a compact list of tags are not aligned
